src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -1295,6 +1298,8 @@ protected UploadResponse upload( InputStream stream, String filepath, long size,
         finally
         {
             closeQuietly( client );
+            closeQuietly( response );
```

Comment:
Can we reverse the order of these two calls? The response should be closed before the client, I think, just in case they share some resources.